### PR TITLE
Update get-started-with-the-default-dlp-policy.md

### LINKED ISSUE #12235: Training Classifiers not available in Business Premium
Please correct your documentation to reflect the fact that training classifiers are not available in BP licensed tenants, thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c11defbb-77c3-c56c-41f2-65e3a4dfafc5
* Version Independent ID: bde5b77f-712d-f811-2315-d775d66fc8a6
* Content: [Get started with the default DLP policy - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/get-started-with-the-default-dlp-policy?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/get-started-with-the-default-dlp-policy.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/get-started-with-the-default-dlp-policy.md)
* Service: **o365-seccomp**
* GitHub Login: @chrfox
* Microsoft Alias: **chrfox**

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about license requirement for classifiers

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12235

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/get-started-with-the-default-dlp-policy.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ To help protect the sensitive information, the default DLP policy:
   - Content containing source code (as matched by our Source Code trainable classifier)
   - Content matching our U.S HIPAA Enhanced template
   - Content containing Intellectual Property (as matched by our IP, Project Documents, M&A Files, Software Product development files, IT Infra and Network Security Documents, and Strategic planning documents trainable classifiers)
+ 
+  > [!NOTE]
+  > Classifiers are a Microsoft 365 E5, or E5 Compliance feature. You must have one of these subscriptions to make use of them.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/get-started-with-the-default-dlp-policy.md]:
```suggestion
  > Trainable classifiers are only available in Microsoft 365 E5 and E5 Compliance. You must have one of these subscriptions to make use of them.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
Closing as per the comment in #12235

**Pull request closed without merging**